### ISSUE TITLE
The packages listed on npmjs.com seem to point to no-where

### ISSUE THREAD

**Issue by grempe** [SpiderOak/crypton#403]
Searching on npmjs.com for crypton leads one to this page:

https://www.npmjs.com/search?q=crypton

It references two packages. Both of which are v0.0.0

SpiderOak has what would appear to be an empty repo for one (should be removed?):
https://github.com/SpiderOak/crypton-cli

If you do an 'npm install' on that package you end up with a bare bones (even working?) CLI that seems to have something to do with integrating with Docker.

The crypton-server package doesn't seem to point anywhere.
If you run 'npm install crypton-server it tries and fails to install what appears to be an ancient version of crypton.

Should'nt they both just be removed from NPM and github?  These rough edges don't reflect well on the project.

**Comment by daviddahl** [comment]
Yes, these should be removed. We dont use those any more - they were 1/2 done projects that I have forgotten about. Thanks for bringing this up

**Comment by daviddahl** [comment]
@ecto do you need to do this removal?

**Comment by grempe** [comment]
Yep, still there...  :-)

**Comment by daviddahl** [comment]
removed!

**Comment by grempe** [comment]
Did you only remove from github?

https://github.com/SpiderOak/?utf8=✓&query=crypton

I still see them here (which is where most people, ahem me, would stumble across them):

https://www.npmjs.com/search?q=crypton

**Comment by daviddahl** [comment]
Ah yes, only github... @ecto needs to remove the npm one as it is under his account
